packages/react-liveness/src/components/FaceLivenessDetector/service/machine/machine.ts

```diff
@@ -373,6 +377,18 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
               1: { target: 'ovalMatching' },
             },
           },
+          handleChallenge: {
+            always: [
+              {
+                target: 'delayBeforeFlash',
+                cond: 'hasFaceMovementAndLightChallenge',
```

Comment:
Nit: would this be more correct as isFaceMovementAndLightChallenge?

Reply:
Yeah I can update that